packages/core/src/shared/utilities/programUtils.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getLogger } from '..'
+import { ChildProcess, ChildProcessOptions } from './processUtils'
+
+/**
+ * Tries to execute a program at path `p` with the given args and
+ * optionally checks the output for `expected`.
+ *
+ * @param p path to a program to execute
+ * @param args program args
+ * @param doLog log failures
+ * @param expected output must contain this string
+ */
+export async function tryRun(
```

Comment:
please leave it in pathFind.ts

it's not a goal for everything in shared/ to be named fooUtils. pathFind.ts was intentionally an attempt to avoid the "fooUtil" pattern where possible, to have more meaningful module names.

Reply:
oh, i just saw this:

> I found another way to stub/spy the method that significantly reduces churn in pathFind.ts. If we move `tryRun` to another file, and import it as a module, we can stub it in the tests.

in that case, let's name the new module pathFind2.ts. no point in pretending that it is something else.